File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/cmd/entity/e40/E40RTKModeEntity.java
package com.shmedo.configlibrary.iot.cmd.entity.e40;

import com.shmedo.configlibrary.ble.interfaces.Validater;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2/25/21 <br/>
 * 描述：      生成GNSS RTK参数拼接指令
 */
public class E40RTKModeEntity implements Validater {
    private String mode;//0表示基站，1表示移动站

    public void setMode(String mode) {
        this.mode = mode;
    }

    @Override
    public void validate() {

    }

    @Override
    public String toString() {
        StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.append("mode=" + mode);

        return stringBuilder.toString();
    }
}

File: lib_device_base/src/main/java/com/shmedo/lib/device/base/iot_cmd/entity/e40/E40RTKModeEntity.kt

package com.shmedo.lib.device.base.iot_cmd.entity.e40

import com.shmedo.lib.device.base.Validater

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2/25/21 <br></br>
 * 描述：      生成GNSS RTK参数拼接指令
 */
class E40RTKModeEntity : Validater {
    var mode //0表示基站，1表示移动站
            : String? = null

    override fun validate() {}
    override fun toString(): String {
        val stringBuilder = StringBuilder()
        stringBuilder.append("mode=$mode")
        return stringBuilder.toString()
    }
}